optuna/integration/allennlp.py

```diff
@@ -274,6 +278,8 @@ def __init__(
         config_file: str,
         serialization_dir: str,
         metrics: str = "best_validation_accuracy",
+        force: bool = False,
+        file_friendly_logging: bool = False,
         *,
         include_package: Optional[Union[str, List[str]]] = None
```

Comment:
The arguments of `force` and `file_friendly_logging` are placed before `include_package`. Changing the order of arguments affects user code. How about placing them after `include_package`?
```suggestion
        *,
        include_package: Optional[Union[str, List[str]]] = None,
        force: bool = False,
        file_friendly_logging: bool = False,
```

Reply:
Thank you for the suggestion! (I think this change doesn't introduce breaking compatibility since `include_package` is a keyword-only argument. But I agree with you for placing them after `include_package`)